lib/avo/fields/base_field.rb

```diff
@@ -256,6 +256,14 @@ def model_or_class(model)
           "model"
         end
       end
+
+      def name_from_id
+        @id.to_s.humanize(keep_id_suffix: true)
+      end
+
+      def name_from_use_resource
```

Comment:
`use_resource` is not a `BaseField` property. Can we create a `name` method in `HasBaseField` that checks for `use_resource` and uses that or falls back to `super`?

Reply:
That approach is kinda tricky because we always want to give priority to custom name or to localization... We can have the name method overridden in HasBaseField but there we need to check for custom name and translation too so that would be a duplicated method. Check my last approach for name